modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/translator/DeepLClient.java

```diff
@@ -42,6 +41,16 @@
 @Component(immediate = true, service = DeepLClient.class)
 public class DeepLClient {
 
+	public static final String AUTH_KEY = "auth_key";
+
+	public static final String SOURCE_LANG = "source_lang";
+
+	public static final String TARGET = "target";
+
+	public static final String TARGET_LANG = "target_lang";
+
+	public static final String TEXT = "text";
+
```

Comment:
Didn't bchan ask us to inline them?
Anyway, if you don't need to inline them, you need to make these private and rename them to "_AUTH_KEY", etc.

Reply:
@adolfopa Does Inline mean hard corded? Then I'll do that , squash and send PR to liferay-lima again.